randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -1193,10 +1205,10 @@ Extra - room_id: [31]
           Varia Suit
           # Tank all 8 of the X
           Damage Boosts (Intermediate) and Normal Damage ≥ 552
-          # Adept movement could avoid some
+          # Adept movement could avoid some: https://youtu.be/U85aWphpi5s
```

Comment:
cant you also do this with only wave?

Reply:
I would need to do some tests to see if wave-only is viable. I feel like I may have gotten lucky with the 3 ice x that I did take and barely avoided the last one.

Reply:
This is possible with only wave but it's just as annoying/hard as doing this with plasma damageless

Reply:
cool then leave as is